web/styling/positioning-tips/direction-column-reverse.md

```diff
@@ -59,17 +59,6 @@ The four child boxes inside class parent will be displayed **vertically** on top
 
 ![566ed55387abab0c00bccab0.svg](https://img.enkipro.com/5cd4ebde7bebedb1168c64d4f3d8ee61.png)
 
----
-## Practice
-
-Where will the first child be displayed?
-
-???
-
-* Bottom-left of the parent.
-* Top-left of the parent.
-* Bottom-right of the parent.
-* Top-right of the parent.
```

Comment:
maybe we can change it to something like this (kinda simple but I think it would be enough for this insight)

Which of the following elements will be displayed first?

```html
<div class="foobar">
  <p>Child 1</p>
  <p>Child 2</p>
</div>
```

```css
.foobar {
  display: flex;
  flex-direction: column-reverse;
}
```

???

* Child 2
* Child 1